.ci/jenkins/dsl/jobs.groovy

```diff
@@ -198,13 +190,13 @@ setupSpecificBuildChainNightlyJob('native', nightlyJobParamsGetter)
 setupSpecificBuildChainNightlyJob('sonarcloud', nightlyJobParamsGetter)
 setupQuarkusIntegrationJob('quarkus-main', nightlyJobParamsGetter)
 setupQuarkusIntegrationJob('quarkus-branch', nightlyJobParamsGetter)
-setupQuarkusIntegrationJob('quarkus-lts', nightlyJobParamsGetter)
-setupQuarkusIntegrationJob('native-lts', nightlyJobParamsGetter)
+//setupQuarkusIntegrationJob('quarkus-lts', nightlyJobParamsGetter)
```

Comment:
I thought you would like to keep LTS checks because this is the productized ones or ?

Reply:
LTS is now in upstream by default, so we basically don't need to have a special LTS job imho